policies/policy_service.go

```diff
@@ -62,6 +62,7 @@ func (s policiesService) AddDataset(ctx context.Context, token string, d Dataset
 	if err != nil {
 		return Dataset{}, errors.Wrap(ErrCreateDataset, err)
 	}
+	//TODO create a function to retrieve a dataset by id
```

Comment:
Is this part of a different PR?

Reply:
I guess it came when merging the develop to fix the conflicts, sorry